### ISSUE TITLE
Proofread no-mixed-operators

### ISSUE THREAD

**Issue by dandv** [eslint/eslint#8875]
Two typos on that page:

* "Default is the groups for each kind of operators."
* "Now, considers about ... This configure has 2 groups"

I'd fix them, but the entire page isn't super clear to me. I wanted to disable warnings for obvious arithmetic expressions like `a * b - c` (if you don't know those operator priorities, you probably shouldn't be programming), but apparently there's no way to exclude a group of operators - you need to list all others?

**Comment by alberto** [comment]
> if you don't know those operator priorities, you probably shouldn't be programming

Please, keep these judgmental opinions to yourself, they have no place here.

**Comment by dandv** [comment]
@alberto: do you know any programmer who doesn't know that `*` and `/` have higher priority than `+` and `-`?

Anyway, sorry to offend your sensibilities, and let me rephrase: I *personally* never get confused by those four operators and wanted to exclude them from the warnings.
